test/s1/appframework/appframework_test.go

```diff
@@ -911,9 +925,11 @@ var _ = Describe("s1appfw test", func() {
 			config, _ = testenv.GetAppframeworkManualUpdateConfigMap(deployment, testenvInstance.GetName())
 			Expect(strings.Contains(config.Data["Standalone"], "status: off")).To(Equal(true), "Config map update not complete")
 
+			// Verify apps downloaded by init container
```

Comment:
NIT:  missing init container check

Reply:
Yes identified the missed check. Needed to chat with you @tpereirasplunk before adding to understand if this was intentional or not. So we should have a check here right ?